Source/PGDebugViewController.swift

```diff
@@ -44,9 +44,10 @@ public class PGDebugViewController: UIViewController {
         self.customPlistObject = customPlistObject
     }
     
-    convenience init(cellModules: [PGDebuggableData]) {
+    convenience init(cellModules: [PGDebuggableData],path plistPath: String) {
```

Comment:
Can you insert whitespace between every function/method parameters onwards?
Like this:
`convenience init(cellModules: [PGDebuggableData], path plistPath: String)`
`func sample(param1: String, param2: Int, param3: Double)`